src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double area();
+}
```

Comment:
Don't use verbs for class/interface names. Consider renaming the interface to something that represents an entity or a characteristic, such as 'Measurable' or 'AreaComputable'.